### PR TITLE
v1.29 final release-notes draft

### PR DESCRIPTION
**Pull request by fsmunoz**
What type of PR is this:
/kind cleanup
/kind documentation

What this PR does / why we need it:

This PR refactors and fine-tunes all the release-notes for v1.29.0, and adds manual sections:

* Major Themes: link to the upcoming release article
* Known issues: add the issues collected in the umbrella issue.

### CODE REVIEW

**Comment by fsmunoz** [comment]
I've reflected the Known Issue (https://github.com/kubernetes/kubernetes/issues/121886), please take a look
/cc @pacoxu @SergeyKanzhelev

I've added the cloud provider removal to the Urgent Updates, as per our conversation. It is also present in the list, but I've added a paragraph in that sections with content from the mid-cycle blog.

/cc @sftim

File: releases/release-1.29/release-notes/release-notes-draft.md [hunk 1]
```diff
@@ -1,45 +1,57 @@
...
- 
+- In-tree cloud providers are now switched off by default. This change will require that users who are currently using in-tree cloud provider integrations (Azure, GCE, or vSphere) enable external cloud controller managers, or opt in to the legacy integration by setting the associated feature gates to  `false`. For more information about how to enable and run external cloud controller managers, read [Cloud Controller Manager Administration](https://kubernetes.io/docs/tasks/administer-cluster/running-cloud-controller/) and [Migrate Replicated Control Plane To Use Cloud Controller Manager](https://kubernetes.io/docs/tasks/administer-cluster/controller-manager-leader-migration/). ([#117503](https://github.com/kubernetes/kubernetes/pull/117503), [@dims](https://github.com/dims))
```
Comment (sftim) [inline releases/release-1.29/release-notes/release-notes-draft.md:17]: LGTM - and thank you.

Comment (sftim) [inline releases/release-1.29/release-notes/release-notes-draft.md:17]: A nit. We can ship without this amendment (IMO).
```suggestion
- In-tree cloud provider integrations are now switched off by default. This change will require that users who are currently using in-tree cloud provider integrations (Azure, GCE, or vSphere) enable external cloud controller managers, or opt in to the legacy integration by setting the associated feature gates to  `false`. For more information about how to enable and run external cloud controller managers, read [Cloud Controller Manager Administration](https://kubernetes.io/docs/tasks/administer-cluster/running-cloud-controller/) and [Migrate Replicated Control Plane To Use Cloud Controller Manager](https://kubernetes.io/docs/tasks/administer-cluster/controller-manager-leader-migration/). ([#117503](https://github.com/kubernetes/kubernetes/pull/117503), [@dims](https://github.com/dims))
```

**Comment by fykaa** [comment]
lgtm!

**Comment by Priyankasaggu11929** [comment]
/lgtm
/approve

/hold
(Please unhold, when ready to merge)

Thank you everyone! :pray:

**Review by sanchita-07 — CHANGES_REQUESTED** [review]
lgtm!

**Review by sanchita-07 — CHANGES_REQUESTED** [review]
PLEG (Pod Lifecycle Event Generator) feature

**Comment by fsmunoz** [comment]
> PLEG (Pod Lifecycle Event Generator) feature

I've added that to the description, while keeping it out of the title to keep is shorter.

**Comment by fsmunoz** [comment]
Squashed in anticipation of merge (more reviews/fixes are still welcomed though).

**Comment by Priyankasaggu11929** [comment]
re-applying lgtm :slightly_smiling_face: 

/lgtm

**Comment by fsmunoz** [comment]
Added the `SchedulerQueueingHints` issue to Known Issues; wording based on the conversation in the channel, with a link to the PR.

/cc @sanposhiho

**Comment by fsmunoz** [comment]
Squashed for merge, all reviews addressed.

@Priyankasaggu11929 take it a way!

**Comment by Priyankasaggu11929** [comment]
/lgtm

final lgtm :smile: :heart:

**Review by kumarankit999 — COMMENTED** [review]
lgtm!!!

**Comment by Priyankasaggu11929** [comment]
maybe a missed webhook :joy: 

trying again!

/lgtm

**Pull request merged**